maas/client/utils/multipart.py

```diff
@@ -16,7 +16,12 @@
 
 __all__ = ["encode_multipart_data"]
 
-from collections import Iterable, Mapping
+try:
+    # Python <= 3.9
+    from collections import Iterable,Mapping
+except:
+    # Python > 3.9
+    from collections.abc import Sequence
```

Comment:
The import change here doesn't look right, Sequence is unused, others are used

Reply:
yes. it should be:  from collections.abc import Iterable,Mapping